templates/layouts/bootstrap.html.ep

```diff
@@ -137,6 +137,10 @@
           <div id='footer-legal' class="text-center">
               openQA is licensed
               %= link_to 'GPL-2.0' => 'https://github.com/os-autoinst/openQA'
+              % if (app->defaults('current_version')) {
+                  - Version
+                  %=app->defaults('current_version')
+              % }
```

Comment:
you don't need app->defaults - $current_version is set.

But for this you need to make sure to always call defaults(current_version => $version) even with empty values. But this would simplify your code (especially the post-conditionals) anyway